modules/core/04-channel/keeper/timeout.go

```diff
@@ -177,7 +177,7 @@ func (k Keeper) TimeoutExecuted(
 		// the upgrade is aborted and the channel is set to CLOSED.
 		if channel.State == types.FLUSHING {
 			// an error receipt is written to state and the channel is restored to OPEN
-			k.MustAbortUpgrade(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), errorsmod.Wrap(types.ErrTimeoutElapsed, "packet timeout elapsed on ORDERED channel"))
+			k.deleteUpgradeInfo(ctx, packet.GetSourcePort(), packet.GetSourceChannel())
```

Comment:
Should we also write an error receipt? Since the channel closes and the counterparty also will close, I guess it's not needed, but maybe it's worth adding it for signalling of what happened?

Reply:
I'm leaning toward adding a log

Reply:
Agree with adding a log statement to signal the upgrade info has been deleted.

The counterparty may also have upgrade info in state which needs to be cleaned up, right? So if a channel is being closed during an upgrade (due to packet timeout), and we want to close the counterparty channel end with `MsgChannelCloseConfirm` then we should also be doing something like the following in that handler:

```golang
if k.hasUpgrade(ctx, portID, channelID) {
    k.deleteUpgadeInfo(ctx, portID, channelID)
}
```

Reply:
I added the deletion of upgrade info in `ChanCloseConfirm`.